indico/modules/events/editing/models/editable.py

```diff
@@ -17,6 +17,7 @@
 class EditableType(int, IndicoEnum):
     paper = 1
     slides = 2
+    poster = 3
```

Comment:
This needs to go along with an alembic revision that updates the check constraint on the columns using this enum.